starter-code/index.html

```diff
@@ -9,6 +10,18 @@
   <script type="text/javascript" src="./js/index.js"></script>
   <title>Ironhack cart</title>
 </head>
+
 <body>
+  <header id="site-header"> Ironhack Merchandising Shop </header>
+    <div class="all-products">
+      <div>IronbubbleHead</div>
+      <div class="product-price">25</div>
+      <div><p>QTY</p><input class="qty" type="number"/></div>
+      <div>totalPrice</div>
+      <button onclick="deleteItem"></button>Delete</button>
+      <button onclick="calculatePrice"></button class="buttonPrice">Calculate Prices</button>
```

Comment:
A couple of things here, you should not have used onclick as you can use DOM Manipulation to achieve the same with:
```
var calculatePriceButton = document.getElementById('calc-prices-button');
calculatePriceButton.onclick = getTotalPrice;
```
In this case you had to add the Id to the button